packages/next/src/server/config-schema.ts

```diff
@@ -440,6 +440,7 @@ export const configSchema: zod.ZodType<NextConfig> = z.lazy(() =>
         serverComponentsHmrCache: z.boolean().optional(),
         authInterrupts: z.boolean().optional(),
         newDevOverlay: z.boolean().optional(),
+        streamingMetadata: z.boolean().optional(),
```

Comment:
I'm not seeing the experimental warning showing up for this since I think the default value isn't specified